src/main/java/com/amihaiemil/docker/Logs.java

```diff
@@ -53,6 +53,33 @@
      *  the expected one.
      */
     Reader follow() throws IOException, UnexpectedResponseException;
+
+    /**
+     * Show both stdout and stderr logs.
+     * @return New Logs instance.
+     * @throws IOException If something goes wrong.
+     * @throws UnexpectedResponseException If the status response is not
+     *  the expected one.
+     */
+    Logs all() throws IOException, UnexpectedResponseException;
```

Comment:
@bkuzmic Actually, I think the ``all()`` method is not needed. Let's read them all by default and if they want only ``stderr`` or ``stdout`` then they call the respective methods :D